integration-tests/clipper_manager_tests.py

```diff
@@ -8,10 +8,13 @@
 from argparse import ArgumentParser
 cur_dir = os.path.dirname(os.path.abspath(__file__))
 sys.path.insert(0, os.path.abspath('%s/../' % cur_dir))
+sys.path.insert(0, os.path.abspath('%s/util/' % cur_dir))
 import clipper_admin.clipper_manager
 Clipper = clipper_admin.clipper_manager.Clipper
+sys.path.insert(0, os.path.abspath(cur_dir))
```

Comment:
To expand on this a bit: The reason we add the parent directory to the path explicitly is because we don't want to make the entire Clipper repo a Python package. But it's okay for the integration tests subdirectory to be a python package.